moto/resourcegroupstaggingapi/models.py

```diff
@@ -163,6 +164,10 @@ def kafka_backend(self) -> KafkaBackend:
     def sagemaker_backend(self) -> SageMakerModelBackend:
         return sagemaker_backends[self.account_id][self.region_name]
 
+    @property
+    def lexv2_backend(self) -> LexModelsV2Backend:
+        return lexv2models_backends[self.account_id][self.region_name]
```

Comment:
This service is not available in every region @jamarcelin - that's why we're seeing the errors in the CI.

You can just guard this with a simple if-statement, similar to what the `workspaces_backends` above does.